wgengine/magicsock/magicsock.go

```diff
@@ -2533,6 +2533,32 @@ func (c *Conn) SetDERPMap(dm *tailcfg.DERPMap) {
 	c.mu.Lock()
 	defer c.mu.Unlock()
 
+	if debugUseDERPAddr() != "" {
```

Comment:
Nit: I believe it's more idiomatic to save the result in a variable, so that you can use it below without repeating the function call. Something like `if derpAddr := debugUseDERPAddr(); derpAddr != "" {`